routers/api/v1/convert/convert.go

```diff
@@ -205,15 +205,25 @@ func ToDeployKey(apiLink string, key *models.DeployKey) *api.DeployKey {
 
 // ToOrganization convert models.User to api.Organization
 func ToOrganization(org *models.User) *api.Organization {
+	apiURL := setting.AppURL + "api/v1/orgs/" + org.LowerName
+	// hide primary email if API caller isn't user itself or an admin
+	publicRepoCount, _ := models.GetPublicRepositoryCount(org)
 	return &api.Organization{
-		ID:          org.ID,
-		AvatarURL:   org.AvatarLink(),
-		UserName:    org.Name,
-		FullName:    org.FullName,
-		Description: org.Description,
-		Website:     org.Website,
-		Location:    org.Location,
-		Visibility:  org.Visibility.String(),
+		ID:               org.ID,
+		AvatarURL:        org.AvatarLink(),
+		UserName:         org.Name,
+		FullName:         org.FullName,
```

Comment:
`Sanitize()` was used in a previous change. It's not needed here?

Reply:
Yeah, I never made that change, as I wrote in another comment. I agree with you that it is handled in the JSON serialization.